examples/hello_lambda/main.rs

```diff
@@ -0,0 +1,56 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use clap::Parser;
+use serde::Deserialize;
+use std::fmt::Debug;
+use std::sync::Arc;
+
+use cobalt_aws::lambda::{run_message_handler, Error, LambdaContext};
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    run_message_handler(message_handler).await
+}
+
+/// The structure of the messages we expect to see on the queue.
+#[derive(Debug, Deserialize)]
+pub struct Message {
+    pub target: String,
+}
+
+/// Configuration we receive from environment variables.
+///
+/// Note: all fields should be tagged with the `#[clap(env)]` attribute.
```

Comment:
For future, we could consider making our own wrapper or re-implementation of this that gives the precise behaviour we want, without having to remember to annotate each attribute. For future :-)